scripts/installscripts/buster-install-default.sh

```diff
@@ -362,6 +350,28 @@ check_existing() {
                 # append variables to config file
                 echo "EXISTINGuseSounds=$EXISTINGuseSounds" >> "${HOME_DIR}/PhonieboxInstall.conf"
 
+                if [ "$(printf '%s\n' "2.1" "$(cat ~/BACKUP/settings/version-number)" | sort -V | head -n1)" = "2.1" ]; then
```

Comment:
Will this also work in future versions?

Reply:
It should. This just checks if the previous version is older than 2.1 and then gives option to install the new gpio-services.